src/System.Management.Automation/engine/remoting/commands/CustomShellCommands.cs

```diff
@@ -2952,8 +2952,8 @@ public sealed class SetPSSessionConfigurationCommand : PSSessionConfigurationCom
 ";
 
         private const string setSessionConfigurationTimeoutQuotasSbFormat = @"
-function Set-SessionPluginIdleTimeoutQuotas([int] $maxIdleTimeoutms, [int] $idleTimeoutms, [bool] $setMaxIdleTimoutFirst) {{
-    if ($setMaxIdleTimoutFirst) {{
+function Set-SessionPluginIdleTimeoutQuotas([int] $maxIdleTimeoutms, [int] $idleTimeoutms, [bool] $setMaxIdleTimeoutFirst) {{
```

Comment:
oh, these C# strings that represents powershell are the worst for editing :)

This `$setMaxIdleTimeoutFirst` change looks good.